web/index.html

```diff
@@ -21,7 +22,10 @@
 
 <body>
   <script type="module" src="./out-tsc/src/TestgridIndex.js"></script>
-  <testgrid-index></testgrid-index>
```

Comment:
I don't think the <testgrid-index> element should be removed for index.html. Possibly this wants a separate .html file?

Reply:
oh yeah, I was using index html to render the tab-summary, I left it as a comment and brought the original `testgrid-index` component back

Reply:
Sounds good! Probably don't want to leave that long-term, but while we're developing this seems fine.

Reply:
Yeah, we will have a single page app and all of those components will probably be inside of `testgrid-app` component in the future